homeassistant/components/yamaha_musiccast/translations/bg.json

```diff
@@ -23,11 +23,11 @@
             },
             "zone_sleep": {
                 "state": {
```

Comment:
Please revert changes to all language files (except for strings.json & en.json).